### ISSUE TITLE
nav button's text not showing against black page on hover

### ISSUE THREAD

**Issue by Eric-Arellano** [Eric-Arellano/outbound-site_web-dev-honors-project#6]
**what**
The text color property has no effect, even though background-color does. [example](https://www.dropbox.com/s/a5dzdj9d7gwgoy1/Screenshot%202016-04-27%2014.39.55.png?dl=0)

Note this only happens with a page--black background, not a page--white background.

**reproduce**
page--black backgrounds can be found on FAQ and Resources. Hover over the nav button "Connect"

Line 463 of CSS for some reason isn't applying when page--black:

```
 .button--white:hover {
        background-color: #FFFDF9;
        border-color: #FFFDF9;
        color: #151818;
    }
```

**Comment by Eric-Arellano** [comment]
**why**
Think it's a specificity issue. Assigning an ID to the nav button with the exact same color property gets the text color to turn black on hover as intended, just not when a class.
_potential specificity conflicts_
- .page--black sets color to white and background to black
- .page--black a sets color to green
- .page--black .mast a sets color to white
- .button--white:hover sets color to black and background to white

Based off of this, it's a specificity issue with .page--black .mast a, since this has specificity 21 vs. 11

**Comment by Eric-Arellano** [comment]
fixed with commit dd2a780
